x11-themes/experience/experience-3.04.ebuild

```diff
@@ -4,9 +4,8 @@
 EAPI=0
 
 DESCRIPTION="GTK+2 themes which copy and improve the look of XP Luna"
-HOMEPAGE="https://art.gnome.org/themes/gtk2/1058"
-#SRC_URI="https://art.gnome.org/download/themes/gtk2/1058/GTK2-EXperience.tar.gz"
-SRC_URI="http://freshmeat.net/redir/${PN}/50795/url_tgz/${PN}-gtk-${PV}.tar.gz"
+HOMEPAGE="https://web.archive.org/web/20130730053042/https://art.gnome.org/themes/gtk2/1058"
+SRC_URI="mirror://gnome/teams/art.gnome.org/themes/gtk2/GTK2-EXperience.tar.gz"
```

Comment:
same as in in -r1

Reply:
"->" isn't allowed in eapi=0

Reply:
On the other hand the unversioned distfile is a QA issue as well. So how about to leave the SRC_URI for the in -r0 untouched?